ts/features/itwallet/common/components/ItwSkeumorphicCard/CardClaim.tsx

```diff
@@ -41,7 +41,7 @@ export type ClaimDimensions = Prettify<
 export type CardClaimProps = Prettify<
   {
     // A claim that will be used to render its component
-    claim: ParsedCredential[number];
+    claim?: ParsedCredential[number];
```

Comment:
Question: why is the claim optional? Can we add a comment about it?

Reply:
Addressed here https://github.com/pagopa/io-app/pull/6184/commits/929f4f25e89016b505730db06cc7795412acc9d8